apps/login/app/(login)/signup/success/page.tsx

```diff
@@ -0,0 +1,19 @@
+import { Button, ButtonVariants } from "#/ui/Button";
+import Link from "next/link";
+
+type Props = {
+  searchParams: { [key: string]: string | string[] | undefined };
+};
+export default async function Page({ searchParams }: Props) {
+  return (
+    <div className="flex flex-col items-center space-y-4">
+      <h1>Registration successful</h1>
+      <p className="ztdl-p">You are registered.</p>
```

Comment:
Should we check if that is true before rendering the page?
If not show an error page?

Reply:
We don't show the page as we directly load `/accounts`. I've removed it.